### ISSUE TITLE
Namings

### ISSUE THREAD

**Issue by cdupont** [dustin/mqtt-hs#20]
Just a mild annoyance, but the naming is quite confusing:
- this repo is named "mqtt-hs"
- Hackage name is "net-mqtt"
- Module name is "Network.MQTT"

It would nice to harmonize the namings.

**Comment by dustin** [comment]
Yeah, naming is hard.  There's an `mqtt` library in hackage that seems to be abandoned as well as a `mqtt-hs` (I only named the repo this because I work in a lot of languages over time).  It's unfortunate how complicated things are.

I'm curious about the module name, though.  Is `Network.MQTT` not a good name for a `net-mqtt` package?

**Comment by cdupont** [comment]
I think `Network.MQTT` is perfect, it matches well with "net-mqtt". Maybe renaming this repo as "net-mqtt" would work?
